### DISCUSSION TITLE
New community leaders / devs needed

### DISCUSSION THREAD

**Discussion by jonyo** [geodesicsolutions-community/geocore-community#182] [Announcements]
I will be stepping down as the primary maintainer for Geocore Community edition.  It has become clear I just won't have the time needed to put the work in to keep the code progressing or the community going.  There is also a possible conflict of interest at my new workplace, and I didn't want to have even the appearance of a conflict, so I felt it important to hand over the reigns now if anyone is interested.

I've released the last of the fixes in a version 20 final release, I feel it has been tested enough at this point to take it out of beta version, so whoever takes over will be taking over a stable release (well, as stable as can be expected).  I know there were a few site owners still using the software, perhaps now that the first stable version is out a community can start to grow around it again.

Anyone interested in taking over let me know!  If there is not any interest I'll be archiving the project (no new changes / discussion etc) after a few months, but I wanted to give a chance in case there was any interest to keep it going.

cc @vicos59 since you have helped a lot with the project, let me know if you are interested in taking over.

**Comment by emsibanda** [comment]
Hi 

Thank you for all the wonderful work you have done and for keeping this project alive. I am not a coder but I would like to volunteer to help with the administration so that this doesn't die.

**Comment by vicos59** [comment]
I would like to see the project kept alive for the discussion and bug reports , if nothing else. Would also need to do something about the website; Keeping it alive or migrating the info here.

Thanks for the final release. I'll announce it on FB later.

I believe you have my FB and gmail info.

**Comment by hektyc** [comment]
@jonyo thanks for everything you have done this far. I will be happy to step in to help keep the project alive. A couple of things I can do. I will use the current wiki to reproduce it on my end. I will start a community forum and hopefully by the time I get these items in place there will be some interest from developers capable of keeping GeoCE alive and well. I am not a programmer so this is as much as I can offer at the moment.

**Comment by vicos59** [comment]
@jonyo Here is one possibility to migrate Dokuwiki to Github: https://github.com/hoxu/dokuwiki2git

**Comment by blufyremedia** [comment]
Hello Folks,

Many thanks to Jonyo and his continued effort in maintaining this community for GeoCore. 

As some of you may know, myself along with our team, have been involved with the GeoCore platform for a very long time, and continue to support, maintain and build upon the platform for our clients. We have no intention of shifting away from that, and remain available for those that require our services. For those that do need immediate support or assistance, please reach out to connect with us.

With Jonyo needing to shift his attention elsewhere, beyond our normal operations, I plan to take steps to be more involved and ensure the continuity and longevity of this community and community edition.

Vicos59, and for the others already involved/volunteering here, let's continue dialog to ensure we all take appropriate steps forward.

Regards,
Kris
BluFyre Media

**Reply by vicos59** [reply]
Sounds good to me.

Also, I don't see the point of setting up yet more independent support sites as a few others have suggested. All of the tools needed are right here on Github. Of course, people are free to do as they see fit. I just hate to see another situation like we had with the last guy, who from what I can see, became upset by something and closed up the support site over night with little warning and not even leaving the data up in archive mode. More can be accomplished as a team effort. Just my thoughts.

**Reply by hektyc** [reply]
@blufyremedia it is pretty exciting to to hear from you. I actually purchased a couple of GeoCore addons many years ago. I truly thought all 3rd party developers had packed their bags and walked away without looking back. Hopefully more of you become involved because it would be great to bring the community back together. It really is a platform worth the investment.

@vicos59 my intentions were to provide a community built on XenForo so people would have a place where all could join in on conversations, ask for help, post tutorials and maybe become a community once again because not everyone understands how to use GitHub because it has that for developers approach. I don't know what happened with Rob (GeoLancer) or the forum he had started but the version you shared is the first I'm hearing of that. If you visit his old forum the message still remains that a new web space (GeoCoreMax.com) is under construction. I personally don't know if it is actually being developed behind the scenes or not but I thought I remember Rob sharing that he was trying to get past some health issues before he can devote anymore time.

Anyway, one thing I cannot spare is wasted time so if building out a nice modern user manual and community forum isn't something of interest that others would use than I would sure hate to waste my time.

I was under the impression that Jonyo was asking someone to take over hosting and maintaining the user manual because he wanted to completely disconnect from this project due to a conflict with his current employer, maybe I misunderstood what he was saying.

**Reply by vicos59** [reply]
It really doesn't matter what specifically happened with the other site and I'm not here to criticize him, which is why I didn't mention the site name. The point was to illustrate what can happen when one person controls a site and then something unexpectedly happens. Another example is when the company went out of business and everyone was left up a creek.

In the beginning, jonyo and I discussing making the project bus-proof, i.e. if someone in control of the project gets hit by a bus one day, someone else is able to pick up the ball and keep going. Currently, the Facebook page is controlled by 2 people. I have some control over Github, but in worst case the project could be cloned and keep going. jonyo controls the website, but the source code is here on Github. 

In my ideal world, everything would be here on Github and 2 or more accomplished programmers would have admin control over Github, with a few others on board to answer questions and manage the discussions.

Another thing to consider is that the userbase is likely to remain small. The glory days of independently-hosted classifieds sites are behind us with the advent of Craig's list and Facebook marketplace.

Just my opinion. Everyone is free to do as they see fit. I plan to stick around and answer user questions as best I can as long as I have time and remain a user of the code.

**Reply by blufyremedia** [reply]
@vicos59 I agree. Ideally we want to keep everything centralized, as a single point of resource for those that utilize the CE.  

@imxweb Great to hear. We've been around a long time, and continue to be involved. We never went anywhere. Our focus has always been the custom needs of our clients though.

Although well-intentioned, the concern on separate sites, resources and forums, is that it segregates the dialog and community.  GitHub provides the features and tools needed for those that are involved with the edition, including the discussion format such as we are in.  It also permits direct correlation between updates, issues/fixes and discussion, all in one place.

I don't want to see anyone waste time or effort, so I believe the most prudent course of action is solidify the community here first. What comes after that should be an evolution of needs and growth.

Regards,
Kris

**Reply by jonyo** [reply]
About conflict of interest, I should be fine on the sidelines but I think just to remove any appearance of conflict I should not contribute anything material to the software itself or how it works.

My employer is very open source friendly, so since it is open source like I said being on the sidelines is probably ok.  But what the software is able to build could conflict so I should probably not do any coding or contribute to discussions about how the code should work beyond a very high level.

About where the community is, I kind of agree about keeping it on GitHub, for the same reason that it is an independent site, if the owner went away it would continue to operate even if in a"read only" state, and could always be forked if needed.

Or alternatively, another independent site if there is a good one that has free accounts for open source but it's more user friendly.  That said, GitHub had been adding and refining the community aspects a bunch, so would likely only get better as time goes on.

The trouble spot right now is dokuwiki, finding a way to convert to something that can live without hosting needed.  If we can do that, then the website can be converted to use GitHub pages, then it does not rely on any one person hosting it.  I know some solutions have been suggested but I'm not able to try them currently, my personal computer went out so waiting on ordered computer to be delivered, wait time is 3 weeks.

So if we get GitHub pages, (simple html CSS images, nothing running scripts on back end), we control the website.  I think the main issue for beginners on GitHub is just overload.  So maybe we could"wrap" the different areas of GitHub, the website would be the main navigation and could link into the areas in GitHub, if that makes sense.

(On mobile so sorry about typos 😂)

**Reply by blufyremedia** [reply]
Sounds good Jonyo. I'll do what I can to assist you to ensure you are in a comfortable position and avoid any conflict.

I do think the .ORG served well to corral users and aid in the transition away from the old website/company, provide a means of getting legacy information available and usher folks here to Git. The more we can streamline and centralize that, the better, in my opinion. As we continue forward, the legacy information becomes less relevant as well.

Longer term, I think your other idea is also valid - using the website just as a simple navigation hub/wrapper to ease the burden for users of getting to what they need.

Reach out once you are back in action (I sent you an email as well), and I'll carve out some time to assist where needed.

Regards,
Kris

**Reply by vicos59** [reply]
@blufyremedia I posted an announcement on the Facebook page regarding your offer for professional services in case anyone needs support or development above and beyond what can be offered here in the community:

https://www.facebook.com/GeoCoreCE/

**Reply by jonyo** [reply]
Looking at the FB posts, thanks @vicos59 for helping with that aspect of it!  I see some engagement there so that is a good sign.

Side note, it is released under the MIT license now, that is technically a little different than public domain wording used in the FB post.

**Reply by vicos59** [reply]
> it is released under the MIT license now

Fixed!

**Reply by blufyremedia** [reply]
> @blufyremedia I posted an announcement on the Facebook page regarding your offer for professional services in case anyone needs support or development above and beyond what can be offered here in the community:
> 
> https://www.facebook.com/GeoCoreCE/

@vicos59 That is wonderful, thank you for that! We are just wrapping up some scheduling commitments on this end, and then will be diving into things here in more depth.

**Comment by rastoddart** [comment]
I am out of town for a few days but I received a few replies that I will get to when I get back. Among those who replied was Rob aka kiwi. I don't know about "bus proof" ;-) but I think we can come up with something that will be a resource to the small community that uses or has interest in using the script. Cheers and good health to everyone.

**Reply by vicos59** [reply]
If you set up a site and would like it announced on the GeoCore Facebook page, let me know and I'll be happy to take care of that:

https://www.facebook.com/GeoCoreCE/

**Reply by rastoddart** [reply]
Definitely. Whatever we do going forward, we will do together for the benefit of everyone using the script. I will be back in a few days. TC

**Comment by nicolaDorian** [comment]


**Reply by jonyo** [reply]
> if you are in a non profit company i don't think is a legal problem for you JONYO

The issue is non-compete.
